hyperspy/io_plugins/tiff.py

```diff
@@ -27,6 +27,7 @@
     with warnings.catch_warnings():
         warnings.simplefilter("ignore")
         from hyperspy.misc.borrowed.tifffile import imsave, TiffFile
+        warnings.simplefilter("default")
```

Comment:
This line doesn't actually do anything (filters are reset after the `with` block resolves). There is however an interesting bug/feature for python 2 that was resolved for python 3.4: https://bugs.python.org/issue4180